server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndex.java

```diff
@@ -80,4 +83,26 @@ public UserDoc getByLogin(String login) {
     return userDoc;
   }
 
+  /**
+   * Returns the users (at most 3) who are associated to the given SCM account. This method can be used
+   * to detect user conflicts. It never returns null.
```

Comment:
don't need to indicate that it never returns null, the @CheckForNull is done for that. More generally, your method name is clear enough and I would not use any comment. They are useful in the case of a public API.